Makefile

```diff
@@ -12,11 +12,13 @@ python-version:
 
 clean:
 	- rm -rf build
-	- find avb -name '*.so' -delete
-	- find avb -name '*.dylib' -delete
-	- find avb -name '*.pyd' -delete
-	- find avb -name '*.dll' -delete
-	- find avb -name '*.pyc' -delete
+	- find src/avb -name '*.so' -delete
+	- find src/avb -name '*.dylib' -delete
+	- find src/avb -name '*.pyd' -delete
+	- find src/avb -name '*.dll' -delete
+	- find src/avb -name '*.pyc' -delete
+	- rm -f src/avb/_ext.cpp
```

Comment:
I added the removal of this file as it is auto-generated by Cython on every build.